config.yaml

```
@@ -18,6 +18,36 @@ options:
       kubeapi-load-balancer charm. The workers will evenly distribute the load by alternating
       between the IP addresses in this list. For example, if you have 2 IPs and 4 workers,
       each IP will be used by 2 workers.
+  events-context:
```

maybe this should be called `nginx-events-config`
```suggestion
  nginx-events-config:
```